### PR TITLE
Modal: Update the width for l and xl for smaller screens

### LINKED ISSUE #317: Modal exceeds viewport width (740px) with `size="l"`
I was testing the following modal:

```javascript
<Modal
        title="Test Modal"
        isVisible={true}
        onClose={() => null}
        size="l"
        centering={true}
      >
  Placeholder message
</Modal>
```

This led to the following screenshots, where the modal was correctly displayed on desktop but not in the 740px-width viewport where the modal width exceeds the viewport:

Screenshot_2020-05-04 Glints - Career Discovery Development Platform
Screenshot_2020-05-04 Glints - Career Discovery Development Platform(1)

### PR DESCRIPTION
**Pull request by ritikamotwani**
Fixes: #317

### CODE REVIEW

File: src/Display/Modal/ModalStyle.ts [hunk 1]
```diff
@@ -87,7 +87,15 @@ export const ModalContentArea = styled.div<ModalContentAreaProps>`
       `;
     }
   }}
-    
+
+  @media (max-width: ${ScreenSize.tablet}px) {
```
Comment (ritikamotwani) [inline src/Display/Modal/ModalStyle.ts]: Currently
```
@media (max-width: ${ScreenSize.mobileM}px) {
    width: 95vw;
  }
```

This helps for the s and m modal sizes as their width is less than the mobileM and tablet device size.

For the sizes `l` and `xl` this criteria should be followed from tablet as the widths of the two are more than the tablet and mobileL device sizes.

File: src/Display/Modal/ModalStyle.ts [hunk 2]
```diff
@@ -87,7 +87,15 @@ export const ModalContentArea = styled.div<ModalContentAreaProps>`
       `;
     }
   }}
-    
+
+  @media (max-width: 950px) {
```
Comment (ritikamotwani) [inline src/Display/Modal/ModalStyle.ts]: Currently

@media (max-width: ${ScreenSize.mobileM}px) {
    width: 95vw;
  }
This helps for the s and m modal sizes as their width is less than the mobileM and tablet device size.

For the sizes l and xl this criteria should be followed from 950px screen size (as width of xl is 920) as the widths of the two are more than the tablet and mobileL device sizes.

Comment (rosslo) [reply src/Display/Modal/ModalStyle.ts]: Is it possible to just add `max-width: 95vw` at line 54 instead?

Comment (ritikamotwani) [reply src/Display/Modal/ModalStyle.ts]: Nice suggestion, thanks!

**Review by rosslo — APPROVED** [review]

**Review by rosslo — APPROVED** [review]

**Pull request merged**